delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/Main.scala

```diff
@@ -133,6 +134,11 @@ object Main extends BIOApp {
       val logger                          = locator.get[Logger]
       val cluster                         = Cluster(as)
 
+      sys.env.get("MIGRATION_1_7").foreach { _ =>
```

Comment:
Can we check if it's a `boolean` and that its value is `true`? It's useful for the deployment to flip true/false.